server/src/main/java/chat/database/UsersBase.java

```diff
@@ -0,0 +1,69 @@
+package chat.database;
+
+import chat.model.User;
+import org.apache.log4j.Logger;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UsersBase {
```

Comment:
It is better to give name for this class like "UserRepository", also the name for package - repositories

Reply:
++